tsl/test/isolation/expected/cagg_watermark_concurrent_update_1.out

```diff
@@ -0,0 +1,226 @@
+Parsed test spec with 3 sessions
```

Comment:
Is it possible to avoid a variant output for this test?

Reply:
The idea is to check the query plan and see what watermark value is used. Unfortunately, PG 13/14 and PG 15/16 create different query plans. So, we need a variant here.